x-pack/plugins/ml/public/components/influencers_list/influencers_list.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+/*
+ * React component for rendering a list of Machine Learning influencers..
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  EuiToolTip
+} from '@elastic/eui';
+
+import { abbreviateWholeNumber } from 'plugins/ml/formatters/abbreviate_whole_number';
+import { getSeverity } from 'plugins/ml/util/anomaly_utils';
+
+
+export class InfluencersList extends Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  getTooltipContent(maxScoreLabel, totalScoreLabel) {
+    return (
+      <React.Fragment>
+        <p>Maximum anomaly score: {maxScoreLabel}</p>
+        <p>Total anomaly score: {totalScoreLabel}</p>
+      </React.Fragment>
+    );
+  }
+
+  renderInfluencerValue(influencerFieldName, valueData) {
+    const maxScorePrecise = valueData.maxAnomalyScore;
+    const maxScore = parseInt(maxScorePrecise);
+    const maxScoreLabel = maxScore !== 0 ? maxScore : '< 1';
+    const severity = getSeverity(maxScore);
+    const totalScore = parseInt(valueData.sumAnomalyScore);
+    const totalScoreLabel = totalScore !== 0 ? totalScore : '< 1';
+
+    // Ensure the bar has some width for 0 scores.
+    const barScore = maxScore !== 0 ? maxScore : 1;
```

Comment:
nitpick: styleguide suggests to always use parentheses for conditionals, the ternary examples also use parentheses.